### ISSUE TITLE
radio.inputs need to be able to accept conditions like checked/disabled

### ISSUE THREAD

**Issue by helderneves91** [livewire/flux#260]
Hi, lets consider `checked` in this explanation.

Actually we can set `checked` to a radio input for example.

In an edit form, we may need to set `checked` based on a wire:model, like (blade syntax):
```
<flux:radio.group wire:model.live="priority_id">
    @foreach($this->priorities as $priority)
        <flux:radio
                value="{{ $priority->id }}"
                label="{{ $priority->name }}"
                description="{{ $priority->description }}"
                @checked($exam->priority_id === $priority_id)
        />
    @endforeach
</flux:radio.group>
```

I looked to the flux stubs and didn't found an way to set this.

Thanks,
Helder

**Comment by lcallif** [comment]
I am successfully using the following:

:checked="($bomInputs[$loop->index]['labormaterialboth'] === 'Labor')"

**Comment by helderneves91** [comment]
Still think there's a bug.
It works **if** the `<flux:radio.group>` doesn't have `wire:model`.

Take this example, which **isn't** wired.
This will auto check based on condition, so work's as expected.
```
<flux:radio.group>
    @foreach($this->priorities as $key => $priority)
        <flux:radio value="{{ $priority->id }}" label="{{ $priority->name }}" description="{{ $priority->description }}" :checked="((int)$exam->priority_id === (int)$priority->id)" />
    @endforeach
</flux:radio.group>
```

Take this example, which **is** wired.
Won't work... The option won't be checked based on condition.
```
<flux:radio.group wire:model.live="priority_id">
    @foreach($this->priorities as $key => $priority)
        <flux:radio value="{{ $priority->id }}" label="{{ $priority->name }}" description="{{ $priority->description }}" :checked="((int)$exam->priority_id === (int)$priority->id)" />
    @endforeach
</flux:radio.group>
```

@calebporzio , help!

**Comment by helderneves91** [comment]
I can confirm the problem is the value being an integer, if it's a string, works as expected.

```
<flux:radio.group wire:model.live="priority_id">
    @foreach($this->priorities as $key => $priority)
        <flux:radio value="{{ $priority->id }}" label="{{ $priority->name }}" description="{{ $priority->description }}" />
    @endforeach
</flux:radio.group>
```

Doesn't need to have `:checked=""` to conditionally check what input should be checked by default.

**Comment by ju5t** [comment]
@helderneves91 this is solved in #227. You may want to close this one.
